docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

```diff
@@ -33,15 +33,20 @@ If you're using an older version of NorthstarProton, replace the information abo
 
 ## Lutris (Wine)
 
-1. If you don't already have the game downloaded, install the game [from here.](https://lutris.net/games/titanfall-2/)
-2. Download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
-3. Extract all contents of the file to your Titanfall 2 folder
-4. **If you have the game installed on Lutris:** right click _Titanfall 2_ > _Configure_ > _Game Options_ > Set _Executable path_ to _NorthstarLauncher.exe_
-5. **Otherwise:** click the `+` button in the top left > set the name to whatever and _Runner_ to _Wine_ > click on _Game options_ > set _Executable path_ to _NorthstarLauncher.exe_ then save.
+1. If you don't already have the game downloaded, you can use [this Lutris script](https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App).
+2. Install the latest version of Northstar using [FlightCore](../installing-northstar/northstar-installers#geckoeidechse-flightcore), [Viper](../installing-northstar/northstar-installers#0negal-viper), or do it manually
```

Comment:
personally think we should be recommending manual first & foremost over auto installers

Reply:
That part is copied from the Steam part of the guide. If you want I can change it there too though.

Reply:
> personally think we should be recommending manual first & foremost over auto installers

Cringe take >:(

Reply:
> > personally think we should be recommending manual first & foremost over auto installers
> 
> Cringe take >:(

![image](https://user-images.githubusercontent.com/116909003/236630466-c4f1c405-7585-4662-8bc3-2c528aeb7a2c.png)